services/cinema_hall.py

```diff
@@ -0,0 +1,15 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> None:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+        hall_name: str,
+        hall_rows: int,
+        hall_seats_in_row: int) -> None:

```

Comment:
Invalid type hint for return value. It returns CinemaHall.
```suggestion
        hall_seats_in_row: int) -> CinemaHall:

```